freqtrade/rpc/rpc.py

```diff
@@ -135,9 +135,11 @@ def _rpc_trade_status(self) -> List[Dict[str, Any]]:
                 trade_dict = trade.to_json()
                 trade_dict.update(dict(
                     base_currency=self._freqtrade.config['stake_currency'],
-                    close_profit=fmt_close_profit,
+                    close_profit=trade.close_profit or None,
+                    close_profit_perc=fmt_close_profit,
                     current_rate=current_rate,
-                    current_profit=round(current_profit * 100, 2),
+                    current_profit_perc=round(current_profit * 100, 2),
+                    current_profit=current_profit,
```

Comment:
```suggestion
                    current_profit=current_profit,
                    current_profit_perc=round(current_profit * 100, 2),
```
-- cosmetics, to be aligned to close_profit and close_profit_perc above